src/ts2fable.fs

```diff
@@ -993,14 +1033,22 @@ let printModule (lines: ResizeArray<string>) (indent: string) (md: FsModule): un
             sprintf "" |> lines.Add
             sprintf "%stype %s%s =" indent al.Name (printTypeParameters al.TypeParameters) |> lines.Add
             sprintf "%s    %s" indent (printType al.Type) |> lines.Add
+        // | FsType.Variable vb ->
+        //     sprintf "" |> lines.Add
+        //     let vtp = vb.Type |> printType
+        //     sprintf "%slet [<Import(\"*\",\"%s\")>] %s: %s = jsNative" indent vb.Name vb.Name vtp |> lines.Add
```

Comment:
I'm still not sure what the logic is for making [variables be imports](https://github.com/fable-compiler/Fable/issues/1200#issuecomment-340768371). `let [<Import("*","iziToast")>] iziToast: IziToast = jsNative`

Reply:
As I see it, the F# variable would represent the npm package/JS module. It's tricky because in JS/TS the module can be exported in several ways. [In the case of `iziToast` there are three](https://github.com/dolce/iziToast/blob/f8cb88d5828a70f278608927e608b3fb2246fac6/types/index.d.ts#L291-L300): as a global variable, as a default export and as a namespace ( `*`)  export.

```typescript
declare const iziToast: IziToast;

export default iziToast;

export {
    IziToastPosition,
    IziToastTransitionIn,
    IziToastTransitionOut,
    IziToastSettings
};
```

[Ignoring the global declaration](https://github.com/fable-compiler/ts2fable/issues/21#issuecomment-341410711) we could represent it in Fable as:

```fsharp
type IziToast =
   interface end // TODO: Fill interface

[<Import("default", "izitoast")>]
let iziToast: IziToast = jsNative

type IExports =
  abstract IziToastPosition: IziToastPosition
  abstract IziToastTransitionIn: IziToastTransitionIn
  abstract IziToastTransitionOut: IziToastTransitionOut
  abstract IziToastSetting: IziToastSettings

// The -Module suffix is tentative, we can use something
// else to distinguish the default and namespace exports

[<Import("default", "izitoast")>]
let iziToastModule: IExports = jsNative
```